source/guides/network/supernode-enrollment.rst

```diff
@@ -0,0 +1,173 @@
+
+   If you already have some of the node accounts (main, transport, remote, VRF or voting) `you can provide them in the preset file too <https://github.com/nemtech/symbol-bootstrap/blob/main/docs/presetGuides.md#specify-the-nodes-keys>`__.
+
+2. **Create and run the node** using the new preset file:
+
+   Along with the standard node services, this will also download and install the **monitoring agent**.
+
+   .. code-block:: bash
+
+      symbol-bootstrap start -p mainnet -a dual -c supernode.yml
+
+   The ``start`` command will create the node configuration and then launch it.
+
+   .. note:: For test purposes you can use |codename|'s TESTNET using ``-p testnet``.
+
+3. Make sure your node is **sufficiently funded**. Your main account must hold at least **1M** |networkcurrency| :ref:`to be eligible as a supernode <supernode-program>`.
+
+   You can find the node's main account address in the ``target/addresses.yml`` file. 
+
+   On the TESTNET, you can **send tokens** to your node's main account using the `Symbol Faucet <http://faucet.testnet.symboldev.network>`__.
+
+4. **Register the node** by linking its remote, VRF and voting keys:
+
+   .. code-block:: bash
+
+      symbol-bootstrap link --useKnownRestGateways
+
+   This sends a transaction linking the supplemental keys to the main account. This transaction pays a small :doc:`fee <../../concepts/fees>` so make sure your main account has **extra funds** beyond the requirements of the Supernode program.
+
+5. **Enroll in the Supernode program**.
+
+   .. code-block:: bash
+
+      symbol-bootstrap enrolRewardProgram --useKnownRestGateways
+
+   This sends a transaction to the Controller address, which includes the node's public key and its monitoring agent's public URL.
+
+   From this point, the :ref:`reward programs controller <reward-programs-controller>` on the network will monitor the node.
+
+.. _enroll-supernode-manually:
+
+********
+Manually
+********
+
+If you don't want (or cannot) use Symbol Bootstrap you can still enroll your node in the Supernode program manually. These instructions are only available **for Linux**.
+
+First off, make sure your node is up and running (or ready to run). `This guide <https://github.com/nemtech/catapult-server/blob/main/docs/RUNPEERLIN.md>`__ explains how to setup the ``catapult-server`` service, but you will need other services as well to fully configure a node (**Guide coming soon**).
+
+The process requires **installing a monitoring agent** and **announcing a special transaction**. You are going to create a few files, so please create a folder to keep your filesystem tidy.
+
+.. note:: The following instructions are for |codename|'s test network (TESTNET). Once |codename| launches they will be updated for the main network (MAINNET).
+
+1. **Download the node monitoring agent**:
+
+   `Download the Linux binary <https://symbol-node-reward.s3-eu-west-1.amazonaws.com/agent_binary/agent-linux.bin>`__ and make it executable:
```

Comment:
The new url will be https://symbol-node-reward.s3-eu-west-1.amazonaws.com/packages/0.1.0/agent_binary/agent-linux.bin

Can you add the version to a variable?

Reply:
You mean a variable for us, that we can update, or a placeholder for the user so he has to find out what the latest version is and build the right URL? The latest sounds uncomfortable.